src/app/public/modules/definition-list/definition-list.component.ts

```diff
@@ -34,9 +72,65 @@ export class SkyDefinitionListComponent {
  */
   @Input()
   public set defaultValue(value: string) {
-    this.service.defaultValue.next(value);
+    this.definitionListService.defaultValue.next(value);
   }
 
+  @Input()
+  public mode: SkyDefinitionListMode = 'default';
+
+  @Input()
+  public orientation: SkyDefinitionListOrientation = 'vertical';
+
+  public isMobile: boolean = false;
+
+  // Use Observable to allow template to get at content of its grandchildren.
+  public templateStream: Subject<QueryList<SkyDefinitionListContentComponent>> =
+    new Subject<QueryList<SkyDefinitionListContentComponent>>();
+
+  @ContentChildren(SkyDefinitionListContentComponent)
+  private contentComponents: QueryList<SkyDefinitionListContentComponent>;
+
+  @ContentChild(SkyDefinitionListHeadingComponent)
+  private headerComponent: SkyDefinitionListHeadingComponent;
+
+  @ViewChild('definitionListElement', {
+    read: ElementRef,
+    static: true
+  })
+  private elementRef: ElementRef;
+
+  private _labelWidth: string;
+
   constructor(
-    public service: SkyDefinitionListService) { }
+    private adapterSerivce: SkyDefinitionListAdapterService,
+    private changeDetector: ChangeDetectorRef,
+    private definitionListService: SkyDefinitionListService
+  ) { }
+
+  public ngAfterContentInit(): void {
+    if (this.headerComponent) {
+      console.warn(
+        '[Deprecation warning] Do not use the `<sky-definition-list-heading>` component in your template (this will be a breaking change in the next major version release).\n' +
```

Comment:
Just to confirm, using the existing `sky-definition-list-heading` component will still work, correct?

Reply:
```suggestion
        '[Deprecation warning] Do not use the `<sky-definition-list-heading>` component in your template because it will be a breaking change in the next major version release.\n' +
```

Just a small grammar suggestion to avoid the run-on sentence. ... Also, we use `sky-definition-list-heading` in the current demo and code example, but I don't see any changes in this PR to correct this. If we are adding a warning to NOT use that component because of future breaking changes, should we also update the demo and code example so that we aren't demonstrating the use of the component that we don't want people to use?.

Reply:
Oh, and it seems like we should probably add the deprecation warning to the component description in the docs for `SkyDefinitionListHeadingComponent` as well. What do you think?

Reply:
Yes, and its still covered in the visual and unit tests.

Reply:
@blackbaud-johnly I committed your suggestion and updated the demos to no longer include the header.